vms/platformvm/state/state.go

```diff
@@ -425,7 +425,15 @@ type ValidatorWeightDiff struct {
 	Amount   uint64 `serialize:"true"`
 }
 
-func (v *ValidatorWeightDiff) Add(negative bool, amount uint64) error {
+func (v *ValidatorWeightDiff) Add(amount uint64) error {
+	return v.add(false, amount)
+}
+
+func (v *ValidatorWeightDiff) Sub(amount uint64) error {
+	return v.add(true, amount)
+}
+
+func (v *ValidatorWeightDiff) add(negative bool, amount uint64) error {
```

Comment:
i feel like `update(add bool, amount uint64)` is a more descriptive function header but doesn't matter too much

Reply:
or maybe `addOrSubtract`